tests/reg_tests/test_DVGeometryVSP.py

```diff
@@ -232,7 +229,6 @@ def sample_uv(nu, nv):
 
             # get the coordinates
             nNodes = len(uv[0, :])
-            ptVecA = openvsp.CompVecPnt01(geoms[0], 0, uv[0, :], uv[1, :])
```

Comment:
I'm not sure. I adapted these tests from Anil and have less experience with openvsp than he does. I expect if the test still passes we are OK.

Reply:
If you look at the tests in this PR, sometimes everything would pass but there was another commit where the same test failed (and I think both had this line removed), so this might be one of those stochastic things. I'll add it in for now just to trigger another build, but maybe @anilyil has more information.

Reply:
Hm now the error's even bigger at `0.278` :disappointed:

Reply:
That `CompVecPnt01` call is not some internal update routine as far as I know, and can be safely removed. I think the error in parallel wing test is from a bug in VSP, which is documented a bit in #70. I will take a look at the existing tests etc and make my comments there. This blob is fine as is.